### ISSUE TITLE
can it work with webview?

### ISSUE THREAD

**Issue by pangminfu** [ideaismobile/AKParallax-Android#10]
will the ParallaxScrollView work if i have and imageview at the top and webview as the bottom content?

**Comment by ygorbarboza** [comment]
no, it wont. maybe if you change the webview and follow the code on scroll view you can do that
